pom.xml

```diff
@@ -475,8 +474,21 @@
                 <artifactId>janusgraph-es</artifactId>
                 <scope>compile</scope>
                 <version>${janus.version}</version>
+                <exclusions>
```

Comment:
is this because this piece is not needed?

I see you have an entry in dependencyManagement for the rest client. If set that should ensure we use the correct version transitively too -- so you should not need the exclude here (adding excludes gets harder to manage ... though obviously they have their uses!)_

Can you check with 'mvn dependency:tree'. Is there nay difference if you remove the exclude? What is it?
Also are you getting the right versions.

I usually run this on master, and my PR, and then use a side by side compare (I tend to use atom) to verify. This is needed - I can run if required, but suggest not merging the PR until verified

We just need to be careful with dependencies as it's so easy for them

Reply:
OWSAP Scan declared that janusgraph-es is HIGH vulnerability. I opened the link I have attached and after I did the dependency tree to see , you can check it has elastic-search-client 6.6.0 that is why I excluded it like that. After the exclusion it does not appear in the tree anymore. Yes lets verify first and then merged it that is why I put a lot of people in the reviewers list. Some use elastic so may have to test it as well.

Reply:
We are going also to check the OWASP again to see if it is still an issue or not before merging.

Reply:
The first section seems to exclude elasticsearch-rest-client by default when janusgraph-es is used as a dependency (though it could be added explicitly). The latter says that if elasticsearch-rest-client is used, it must be at 6.8.6 -- is this what you are trying to achieve?

Also which CVE/issue are you addressing? I notice that our CLM scans aren't showing an issue with elastic-search-client, other than it's old. 

I'll try running the maven build with 'mvn clean install -DCVE' which should generate the CVE report to double check.

Reply:
https://nvd.nist.gov/view/vuln/search-results?adv_search=true&cves=on&cpe_version=cpe%3A%2Fa%3Aelastic%3Aelasticsearch%3A0.4.0
Hi Nigel, this is the CVE/issue that we are addressing.